src/components/atoms/authors/authors.test.tsx

```diff
@@ -1,13 +1,13 @@
 import { render, screen, fireEvent } from '@testing-library/react';
 import { authors } from '../../../utils/mocks';
+import { createAuthorId } from '../../../utils/create-author-id';
 import { Authors } from './authors';
 
 describe('authors', () => {
   it('should render correctly a list of authors', () => {
-    render(<Authors authors={[authors[0]]}/>);
-    const captainAmerica = screen.getByText('Steve Rogers');
+    const { container } = render(<Authors authors={[authors[0]]}/>);
 
-    expect(captainAmerica).toBeInTheDocument();
+    expect(container.querySelector('.authors-link')!.textContent).toStrictEqual('Steve Rogers');
```

Comment:
This was necessary to protect against the introduce of whitespace before or after the author name.

See: https://github.com/elifesciences/enhanced-preprints-client/actions/runs/4377362358/jobs/7660739277